example/src/NestedDynamicText.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import { useRef } from 'react';
+
+import { SafeAreaView, ScrollView, StyleSheet } from 'react-native';
+import { TextInput } from 'react-native-gesture-handler';
+import Rive, {
+  Alignment,
+  Fit,
+  RNRiveError,
+  RNRiveErrorType,
+  RiveRef,
+} from 'rive-react-native';
+
+export default function NestedDynamicText() {
+  const riveRef = useRef<RiveRef>(null);
+
+  const handleInputChange = (e: string) => {
+    // Set the TextRun value of the 'name' TextRun
+    // The name must exist else an error will be thrown
+    // See: https://help.rive.app/runtimes/text
```

Comment:
Correct link: https://rive.app/community/doc/text/docn2E6y1lXo#readupdate-nested-text-runs-at-runtime

`help.rive` is the old docs